targets/cpp-ue4/make.js

```diff
@@ -8,23 +7,29 @@ if (typeof (templatizeTree) === "undefined") templatizeTree = function () { };
 
 var maxEnumSize = 255;
 
+var copyright =
+`//////////////////////////////////////////////////////
+// Copyright (C) Microsoft. 2018. All rights reserved.
+//////////////////////////////////////////////////////
+`;
+
 exports.makeCombinedAPI = function (apis, sourceDir, apiOutputDir) {
     var locals = {
         apis: apis,
         buildIdentifier: exports.buildIdentifier,
+        copyright: copyright,
         errorList: apis[0].errorList,
         errors: apis[0].errors,
         friendlyName: "PlayFab Cpp Sdk",
         sdkVersion: exports.sdkVersion,
-        ueTargetVersion: "4.19"
+        ueTargetVersion: "4.19.0"
```

Comment:
There was an important reason why this wasn't specified.
I think it gives you a lot of warnings if you have 4.19.1, and you're loading a 4.19.0 plugin.
It's not a blocker, but it's a bad customer experience.

Reply:
I actually get warnings if it only specifies 4.19 and not 4.19.0. FWIW, the marketplace guidelines indicate the EngineVersion is supposed to be 4.x.0, not just 4.x. https://www.unrealengine.com/en-US/marketplace-guidelines